script/deploy/safe/safe-decode-utils.ts

```
@@ -7,6 +7,164 @@
+        try {
+          const abiData = JSON.parse(fs.readFileSync(jsonPath, 'utf8'))
+          if (abiData.abi) {
+            // For now, we'll return the first valid ABI we find
+            // In a more sophisticated implementation, we could try to match by bytecode or other means
+            const contractNameFromFile = jsonFile.replace('.json', '')
+            consola.info(`Found potential local ABI: ${contractNameFromFile}`)
+            return { abi: abiData.abi, name: contractNameFromFile }
+          }
+        } catch (error) {
+          // Skip invalid JSON files
+          continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)